students/14th/team9/giyong-gim/user/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class User(models.Model):
+    username     = models.CharField(max_length       = 30)
+    email        = models.EmailField(max_length      = 50)
+    password     = models.CharField(max_length       = 100)
+    phone_number = models.CharField(max_length       = 30)
+    user_created = models.DateTimeField(auto_now_add = True)
```

Comment:
앞 부분은 `=` 을 기준으로 정렬해주었을때 가독성이 좋아지지만, 뒤쪽 kwargs 부분들은 전부 동일하게 정렬해주면 가독성이 좋지 않습니다. 
```suggestion
    username     = models.CharField(max_length = 30)
    email        = models.EmailField(max_length = 50)
    password     = models.CharField(max_length = 100)
    phone_number = models.CharField(max_length = 30)
    user_created = models.DateTimeField(auto_now_add = True)
```

Reply:
수정했습니다.